cspell.config.js

```diff
@@ -58,6 +60,16 @@ const config = {
             path: './project-words.txt',
             noSuggest: true,
         },
+        {
```

Comment:
I think we can just add these terms to the `project-words.txt`. No need to add more tags.

The data directory is generally for data pulled in automatically from external sources, rather than manually written.